cg/constants/pacbio.py

```diff
@@ -0,0 +1,19 @@
+"""Constants related to PacBio sequencing."""
+
+
+class PacBioDirsAndFiles:
+    ADAPTER_REPORT: str = "adapter.report.json"
+    CCS_REPORT: str = "ccs.report.json"
+    CONTROL_REPORT: str = "control.report.json"
+    LOADING_REPORT: str = "loading.report.json"
+    RAW_DATA_REPORT: str = "raw_data.report.json"
```

Comment:
What do you think of these names? @Vince-janv 
```suggestion
    HIFI_REPORT: str = "ccs.report.json"
    CONTROL_REPORT: str = "control.report.json"
    PRODUCTIVITY_REPORT: str = "loading.report.json"
    POLYMERASE_REPORT: str = "raw_data.report.json"
```

Reply:
I think the only one I want to change is css. I found this:

> ccs combines multiple subreads of the same SMRTbell molecule using a statistical model to produce one highly accurate consensus sequence, also called a HiFi read, along with base quality values. This tool powers the Circular Consensus Sequencing workflow in SMRT Link for Sequel and Sequel II platform. On Sequel IIe and Revio platforms, HiFi generation is perfomed on the instrument.

From that, I vote for the name `BASE_CALLING_REPORT`

Reply:
fixed